tcf_website/templates/base/sidebar.html

```diff
@@ -4,6 +4,15 @@
 <div id="sidebar-wrapper" class="border-right pb-0
      d-flex flex-column justify-content-between text-center">
   <ul class="list-group list-group-flush mt-3">
+    <li>
```

Comment:
What do you think about the order of the links on the sidebar? I think putting `Blog` between `Review` and `About` might be good